runtime/integration-tests/src/generic/cases/ethereum_transaction.rs

```diff
@@ -40,9 +40,12 @@ fn call<T: Runtime>() {
 		utils::evm::mint_balance_into_derived_account::<T>(creator_address, 1 * CFG);
 		utils::evm::mint_balance_into_derived_account::<T>(sender_address, 1 * CFG);
 
-		let contract_address = utils::evm::deploy_contract::<T>(
-			creator_address,
-			hex::decode(TEST_CONTRACT_CODE).unwrap(),
+		let contract_address = sp_core::H160::from(
+			utils::evm::deploy_contract::<T>(
```

Comment:
NIT: `deploy_contact()` already returns a `H160`

Reply:
tech debt. Before your changes for 1.7.2 the version of `sp_core::H160` and `ethereum_types::H160` pointed to different versions of the same `H160`, making it necessary to convert them all the time. I will try to clean that up ^^

Reply:
Uggh, I think that has been solved after cleaning the dependency tree

Reply:
Tried fixing it here: https://github.com/centrifuge/centrifuge-chain/pull/1720/commits/e4e048a574e393061f0169e88d2d66b7e1970760